src/layouts/index.tsx

```diff
@@ -54,9 +55,9 @@ export default function Layout({ children }: LayoutProps): JSX.Element {
           height: '100vh',
         }}
       >
-        <header>{/* TODO */}</header>
+        <Navbar />
 
-        <main sx={{ flex: 1 }}>{children}</main>
+        <main /* sx={{ flex: 1 }} */>{children}</main>
```

Comment:
Ezt miért kommentezted ki? Azért van rá szükség, hogy a footer kevés tartalom esetében is [az oldal aljához tapadjon](https://css-tricks.com/couple-takes-sticky-footer/).